catalog-rest-service/src/main/resources/json/schema/metadataIngestion/databaseServiceMetadataPipeline.json

```diff
@@ -25,6 +26,43 @@
       },
       "additionalProperties": false,
       "required": ["dbtCatalogFilePath", "dbtManifestFilePath"]
+    },
+    "dbtHttpConfig": {
+      "title": "DBT HTTP Config Source",
+      "description": "DBT Catalog and Manifest HTTP path configuration.",
+      "type": "object",
+      "properties": {
+        "dbtCatalogHttpPath": {
+          "description": "DBT catalog file to extract dbt models with their column schemas.",
```

Comment:
The description here is same as `dbtCatalogFilePath `, is this intentional?

Reply:
Yes the description of both httppath and localpath is same since they are directly paths to the respective files